### ISSUE TITLE
[EN][Brewfest]Extra Brewfest Prize Token

### ISSUE THREAD

**Issue by kharn227** [Atlantiss/BugTracker#4541]
"This isn't exactly a quest, but is a very good option for accumulating more  Brewfest Prize Token. This "quest" is available after successfully completing There and Back Again. Simply speak to the NPC again and offer to help him out again. The questgiver will give you another ram with which you must retrieve and return more kegs, awarding 2  Brewfest Prize Token per loop and 30 additional seconds. You can do this as many times as possible until your ram mount expires at 15 minutes. You can only offer to help him once every 18 hours, and only after completing the initial quest."
Quote from wowhead. Has no couldown. Gives no tokens per keg.

**Comment by xolon** [comment]
Fix this pl0x

**Comment by Stephenjawn33** [comment]
Yeah, I really hope both the 30 seconds increase on ram timer per round increase, and the 2 tokens per round also gets increased. Would help a lot with the daily tokens.

**Comment by aendry** [comment]
bump
